components/hab/src/command/origin/rbac/show_role.rs

```diff
@@ -0,0 +1,64 @@
+use crate::{api_client::{self,
+                         Client},
+            common::ui::{Status,
+                         UIWriter,
+                         UI},
+            error::{Error,
+                    Result},
+            PRODUCT,
+            VERSION};
+use habitat_core::{origin::Origin,
+                   util::text_render::PortableText};
+use reqwest::StatusCode;
+use url::Url;
+
+pub async fn start(ui: &mut UI,
+                   bldr_url: Url,
+                   origin: Origin,
+                   token: &str,
+                   member_account: &str,
+                   to_json: bool)
+                   -> Result<()> {
+    let api_client = Client::new(bldr_url, PRODUCT, VERSION, None).map_err(Error::APIClient)?;
+
+    match api_client.get_member_role(origin.clone(), token, member_account)
+                    .await
+    {
+        Ok(resp) => {
+            if to_json {
+                match resp.as_json() {
+                    Ok(body) => {
+                        println!("{}", body);
+                        Ok(())
+                    }
+                    Err(e) => {
+                        ui.fatal(format!("Failed to deserialize into json! {:?}.", e))?;
+                        Err(Error::from(e))
+                    }
+                }
+            } else {
+                ui.status(Status::Discovering, "origin member role".to_string())?;
+                println!("Member {} has the '{}' role in the {} origin.",
```

Comment:
Should this be a `println`? Maybe a `ui.status` or `debug!` instead?

Reply:
Added `ui.status` in https://github.com/habitat-sh/habitat/pull/7841/commits/78f9d5e6a2b76e641bb6f2a2971c62744b5e762b